UI.tsv

```diff
@@ -1,14 +1,14 @@
-﻿UI_20150317_000001	{@st41b}Learn Attributes{nl}from Masters of each class.{/}
-UI_20150317_000002	{@st43}List {/}
-UI_20150317_000003	{@st59}Close {/}
+UI_20150317_000001	{@st41b}Learn Attributes{nl}from Masters of each class.{/}
```

Comment:
{@st41b}Learn Attributes{nl}from Masters of each class.{/}
-> {@st41b}Learn Attributes{nl}from the Masters of each class.{/}

Sounds better to me, what do you think?